ui/src/components/shared/Router.tsx

```diff
@@ -47,6 +61,21 @@ export function Router() {
         : acc;
     }, []);
 
+  useEffect(() => {
+    if (env.warningMessage && isShowingWarning) {
```

Comment:
I'd not show the notification on all the pages, it's not an alert for the QR code pages, I'd move it to SiderLayoutContent at this moment

Reply:
Yep, I was about to comment the same on this. My proposal was to add this as a prop per-route. I don't like basing this on the layout either.

Reply:
Good point!

Reply:
I think it makes sense to alert a holder that this is a test environment and more importantly that data will be deleted?

Reply:
This is re-implemented according to this [discussion](https://0xpolygon.slack.com/archives/C044J4HN360/p1683278628642289).